zenodo/modules/pages/utils.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2015 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Utils for communities."""
+
+from __future__ import absolute_import, print_function
+
+from flask import current_app
+from flask_mail import Message
+from invenio_mail.tasks import send_email
+
+from .config import EMAIL_BODY_TEMPLATE, EMAIL_TITLE_TEMPLATE, SENDER_EMAIL, \
+    SUPPORT_EMAIL
+
+
+def render_template_to_string(template, context):
+    """Render a template from the template folder with the given context.
+
+    Code based on
+    `<https://github.com/mitsuhiko/flask/blob/master/flask/templating.py>`_
+    :param template: the string template, or name of the template to be
+                     rendered, or an iterable with template names
+                     the first one existing will be rendered.
+    :param context: the variables that should be available in the
+                    context of the template.
+    :return: a string.
+    """
+    template = current_app.jinja_env.get_or_select_template(template)
+    return template.render(context)
+
+
+def format_request_email_title(context):
+    """Format the email message title for contact form notification.
+
+    :param context: Context parameters passed to formatter.
+    :type context: dict.
+    :returns: Email message title.
+    :rtype: str
+    """
+    template = EMAIL_TITLE_TEMPLATE,
```

Comment:
Yes, should do the same for redirector would but redirector does not define a flask extension (ext.py), that's why there's no convention on how to initialize config properly.